src/query-builder/function-module.ts

```diff
@@ -39,52 +38,13 @@ import { createQueryId } from '../util/query-id.js'
  * having count("pet"."id") > $1
  * ```
  */
-export class FunctionBuilder<DB, TB extends keyof DB> {
+export class FunctionModule<DB, TB extends keyof DB> {
```

Comment:
I think you can now remove the `#oneArgFunction` method.

Reply:
I'm confused, I swear I did this.. it doesn't exist on my end. 😕